src/lib/y2storage/fake_device_factory.rb

```diff
@@ -46,12 +46,13 @@ class FakeDeviceFactory < AbstractDeviceFactory
       {
         "disk"       => ["partition_table", "partitions", "file_system"],
         "partitions" => ["partition", "free"],
-        "partition"  => ["file_system"],
+        "partition"  => ["file_system", "encryption"],
+        "encryption" => ["file_system"],
         "lvm_vg"     => ["lvm_lvs", "lvm_pvs"],
         "lvm_lvs"    => ["lvm_lv"],
-        "lvm_lv"     => ["file_system"],
+        "lvm_lv"     => ["file_system", "encryption"],
         "lvm_pvs"    => ["lvm_pv"],
-        "lvm_pv"     => []
+        "lvm_pv"     => ["encryption"]
```

Comment:
Is this right? Documentation says "For LVM PVs, add this to the corresponding partition"